pandas/tests/window/test_rolling.py

```diff
@@ -696,3 +696,21 @@ def scaled_sum(*args):
     expected = DataFrame(data={"X": [0.0, 0.5, 1.0, 1.5, 2.0]}, index=_index)
     result = df.groupby(**grouping).rolling(1).apply(scaled_sum, raw=raw, args=(2,))
     tm.assert_frame_equal(result, expected)
+
+
+def test_rolling_numerical_accuracy_kahan():
+    # GH: 36031 implementing kahan summation
```

Comment:
can you add the other test from the OP (the succeeding one) as well, pls parameterize.

Reply:
Done